test/seed.spec.js

```diff
@@ -2403,14 +2403,14 @@ describe("> Contract: Seed", () => {
 
           await expect(
             Seed_shortTipVesting.retrieveSeedTokens({
-              refundReceiver: admin.address,
+              refundReceiver: treasury.address,
             })
```

Comment:
Seems like you changed the function to not take `refundReceiver` anymore, but in the tests that prop is still passed

![image](https://user-images.githubusercontent.com/30693990/203311127-9a0a2e23-5805-4df8-97c5-60fac8a50e47.png)